### ISSUE TITLE
View support

### ISSUE THREAD

**Issue by SirParser** [brink-daniel/ssms-object-explorer-menu#2]
I can assign a new menu item to Server/Database/View context, but the query doesn't replace the placeholders.
Are there any plans to support Views?

**Comment by brink-daniel** [comment]
Yes, absolutely I can do that. I'll need to update the code to support it. I'm just on a few other projects this week. Will let you know when it is done.

**Comment by brink-daniel** [comment]
Support has been added views in Version 2.2. Please use tag {VIEW}

**Comment by SirParser** [comment]
Thanks Daniel,
but something went wrong - see attached screenshot.
![SSMS](https://github.com/user-attachments/assets/7612a9f5-e986-4285-af0b-cfa70ae0bbd0)
I simply replaced the files of the old version.
After reverting back to v. 2.1.1 it works OK.

**Comment by SirParser** [comment]
ActivityLog.xml error
[SSMS.txt](https://github.com/user-attachments/files/17716755/SSMS.txt)

**Comment by brink-daniel** [comment]
This is a known issue which will be fixed when I create a proper installer for the extension. For now, please the the FAQ on unblocking the zip file before extracting it.

**Comment by SirParser** [comment]
Thanks Daniel, it's not the first time I scratch my head because of this 'feature'. Most likely not the last one.😆
Thanks for your efforts.
